src/lib/utils.test.ts

```diff
@@ -405,6 +407,21 @@ describe('utils', () => {
         assert.strictEqual(encodeArrayKey(key, index), expectedValue);
       });
     });
+
+    it('should encode the array length only if passing a number', async () => {
+      const schema: ERC725JSONSchema = {
+        name: 'LSP3IssuedAssets[]',
+        key: '0x3a47ab5bd3a594c3a8995f8fa58d0876c96819ca4516bd76100c92462f2f9dc0',
+        keyType: 'Array',
+        valueContent: 'Address',
+        valueType: 'address',
+      };
+
+      const decodedValue = 3;
+      const encodedValue = '0x00000000000000000000000000000003';
+
+      assert.equal(encodeKey(schema, decodedValue), encodedValue);
+    });
```

Comment:
what is the reason for putting this `it` under the `describe('encodeArrayKey',` describe?

The function being tested is `encodeKey` so I suggest to put it under the corresponding "describe" (prob `describe('encodeKey/decodeKey', () => {`)